core/templates/domain/utilities/browser-checker.service.ts

```diff
@@ -61,6 +61,30 @@ export class BrowserCheckerService {
   isMobileDevice(): boolean {
     return this._isMobileDevice();
   }
+
+  /**
+   * Detects the type of browser from it's userAgent.
+   *
+   * @returns {string} - The name of the browser that is being used.
+   */
+  detectBrowserType(): string {
+    const userAgent = this.windowRef.nativeWindow.navigator.userAgent;
+
+    if (userAgent.includes('edg') || userAgent.includes('Edge')) {
```

Comment:
I suggest adding a comment here mentioning that the returned browser names shouldn't be changed since they're directly used for filtering platform parameters.